.env.sample

```diff
@@ -0,0 +1,2 @@
+GRAPHQL_ENDPOINT=xxx
+API_KEY=xxx
```

Comment:
I'd say it's ok to add real usable values rather than adding dummy data, only in the case of local development.
`.env` worked with Next.js in my local machine.

Reply:
I understand your point about using real values in .env for local development. However, it's crucial to ensure these values remain confidential.
Maybe we should continue using placeholders or dummy data in .env files, especially when sharing or committing code, to keep sensitive details secure.

What are your thoughts on maintaining this level of security? 🤔

Reply:
As long as I google/check how it works, supabase cli generates same credentials across developer environments.

And this anon key comes from below config. 
https://github.com/supabase/cli/blob/afd67b3ef923f0bf98821174ee3aaf4efc490137/internal/utils/config.go#L158

So maintaing this `default credentials` with git which in and of itself is not gonna cause crucial problem.

But to avoid adding `prod credentials` in .env unintentionally, not to maintain this credentials makes sense.
(In that case, we might better add git-secrets though..)